.env.example

```diff
@@ -1,3 +1,15 @@
 DEFAULT_LOGGING_LEVEL=info
 APP_LOGGING_LEVEL=debug
-WEB_LOGGING_LEVEL=debug
\ No newline at end of file
+WEB_LOGGING_LEVEL=debug
+
+PM_ORACLE_HOST=localhost
+PM_ORACLE_PORT=1523
+PM_ORACLE_DATABASE_NAME=pm
+PM_ORACLE_USERNAME=test
+PM_ORACLE_PASSWORD=test
+
+MONGO_HOST="pagopa-ecommerce-mongo"
+MONGO_PORT=27017
+MONGO_USERNAME=admin
+MONGO_PASSWORD=password
+MONGO_SSL_ENABLED=false
```

Comment:
Add these to README :)

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/6/commits/e0eb167c0a6c70d1fa55908a50cdd9d7d0d30264